src/Button.js

```diff
@@ -7,6 +7,8 @@ class Button extends React.Component {
         accent: PropTypes.bool,
         className: PropTypes.string,
         colored: PropTypes.bool,
+        component: PropTypes.any,
+        href: PropTypes.any,
```

Comment:
I think we should not restrict here too. What if there's some component which use a prop called `href` and it can be function or node or something.

For example, what if I'll write some `ScrollToElement` component which will scroll to some element. It can pass function to return element id or something like this.

It's strange case though. What do you think?